aws/logs_monitoring/README.md

```diff
@@ -9,25 +9,25 @@ aliases:
   - /serverless/libraries_integrations/forwarder/
 ---
 
+## Overview
+
 The Datadog Forwarder is an AWS Lambda function that ships logs from AWS to Datadog, specifically:
 
-- Forward CloudWatch, ELB, S3, CloudTrail, VPC, SNS, and CloudFront logs to Datadog
-- Forward S3 events to Datadog
-- Forward Kinesis data stream events to Datadog (only CloudWatch logs are supported)
-- Forward metrics, traces, and logs from AWS Lambda functions to Datadog. **Note**: Datadog recommends that you use the [Datadog Lambda Extension](https://github.com/DataDog/datadog-lambda-extension) to monitor your Lambda functions.
+- Forward CloudWatch, ELB, S3, CloudTrail, VPC, SNS, and CloudFront logs to Datadog.
+- Forward S3 events to Datadog.
+- Forward Kinesis data stream events to Datadog (only CloudWatch logs are supported).
+- Forward metrics, traces, and logs from AWS Lambda functions to Datadog. Datadog recommends you use the [Datadog Lambda Extension][1] to monitor your Lambda functions.
 
-For additional information on sending AWS services logs with the Datadog Forwarder, read the [Send AWS Services Logs with the Datadog Lambda Function](https://docs.datadoghq.com/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/) guide.
+For serverless customers using the Forwarder to forward metrics, traces, and logs from AWS Lambda logs to Datadog, you can [migrate to the Datadog Lambda Extension][3] to collect telemetry directly from the Lambda execution environments. The Forwarder is still available for use, but will only receive security updates.
```

Comment:
The forwarder is still very much being developed for all other AWS logs integrations, so don't want to saw "only receive security updates"

```suggestion
For serverless customers using the Forwarder to forward metrics, traces, and logs from AWS Lambda logs to Datadog, you should [migrate to the Datadog Lambda Extension][3] to collect telemetry directly from the Lambda execution environments. The Forwarder is still available for use in Serverless monitoring, but will not be updated to support the latest features.
```